.run/JDBC ignite3 load workloada (127.0.0.1).run.xml

```diff
@@ -0,0 +1,16 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="JDBC ignite3 load workloada (127.0.0.1)" type="Application" factoryName="Application" folderName="Scenarios">
+    <option name="MAIN_CLASS_NAME" value="site.ycsb.Client" />
+    <module name="ignite3-binding" />
+    <option name="PROGRAM_PARAMETERS" value="-load -db site.ycsb.db.ignite3.IgniteJdbcClient -P ./workloads/workloada -p hosts=&quot;127.0.0.1&quot; -s -threads 1" />
```

Comment:
At the moment it is better to use `workloadc` everywhere. `workloadc` is a 100% read workload which we use to measure `read` operations in the `run` mode. To measure `put` operations we use `workloadc` + `load` mode.

Reply:
updated